modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APISchemaRelevantObjectEntryModelListenerTest.java

```diff
@@ -48,7 +48,9 @@ public void test() throws Exception {
 			JSONUtil.put(
 				"applicationStatus", "unpublished"
 			).put(
-				"baseURL", RandomTestUtil.randomString()
+				"baseURL",
```

Comment:
@jalber786 we normally [don't do chaining](https://github.com/liferay/liferay-portal/blob/53c9e2adf7ef70866216d0d3eb48c4bef0fa0939/modules/util/source-formatter/documentation/chaining.markdown) (Brian will likely reject it). Instead do `StringUtil.toLowerCase(RandomTestUtil.randomString()`.